src/popup.tsx

```diff
@@ -30,87 +30,105 @@ import { NavigationBar } from "~components/popup/Navigation";
     syncLabels();
+
+    // check expanded view
+    if (new URLSearchParams(window.location.search).get("expanded")) {
+      setExpanded(true);
+    }
   }, []);
 
   return (
     <Provider theme={theme}>
       <HardwareWalletTheme>
         <GlobalStyle />
-        <HideScrollbar />
-        <Wrapper>
-          <Router hook={useHashLocation}>
-            <HistoryProvider>
-              <Route path="/" component={Home} />
-              <Route path="/purchase" component={Purchase} />
-              <Route path="/confirm-purchase" component={ConfirmPurchase} />
-              <Route path="/purchase-pending" component={PendingPurchase} />
-              <Route path="/receive" component={Receive} />
-              <Route path="/send/transfer/:id?">
-                {(params: { id?: string }) => <Send id={params?.id} />}
-              </Route>
-              <Route path="/send/auth/:tokenID?">
-                {(params: { tokenID: string }) => (
-                  <SendAuth tokenID={params?.tokenID} />
-                )}
-              </Route>
-              <Route path="/explore" component={Explore} />
-              <Route path="/unlock" component={Unlock} />
-              <Route path="/tokens" component={Tokens} />
-              <Route path="/token/:id">
-                {(params: { id: string }) => <Asset id={params?.id} />}
-              </Route>
-              <Route path="/collectibles" component={Collectibles} />
-              <Route path="/collectible/:id">
-                {(params: { id: string }) => <Collectible id={params?.id} />}
-              </Route>
-              <Route path="/transaction/:id/:gateway?">
-                {(params: { id: string; gateway?: string }) => (
-                  <Transaction id={params?.id} gw={params?.gateway} />
-                )}
-              </Route>
-              <Route path="/send/confirm/:token/:qty/:recipient/:message?">
-                {(params: { token: string; qty: string }) => (
-                  <Confirm
-                    tokenID={params?.token}
-                    qty={Number(params?.qty || "0")}
-                  />
-                )}
-              </Route>
-              <Route path="/send/recipient/:token/:qty/:message?">
-                {(params: { token: string; qty: string; message?: string }) => (
-                  <Recipient
-                    tokenID={params?.token}
-                    qty={Number(params?.qty || "0")}
-                    message={params?.message || ""}
-                  />
-                )}
-              </Route>
-              <NavigationBar />
-            </HistoryProvider>
-          </Router>
-        </Wrapper>
+        <HideScrollbar expanded={expanded} />
+        <ExpandedViewWrapper>
+          <Wrapper expanded={expanded}>
+            <Router hook={useHashLocation}>
+              <HistoryProvider>
+                <Route path="/" component={Home} />
+                <Route path="/purchase" component={Purchase} />
```

Comment:
could we pass `expanded` to the pages to disable the variants/animations in Route.tsx?

Reply:
Sure!
I was actually wondering if we could change from a motion effect to a blur effect or something similar. At least in expanded view. Since I didn't have that much time, I pushed it as it was.

Reply:
sounds good! but yea, we can keep it minimal for now and look to implement another effect later on

Reply:
I pushed a small change to fix this. lmk if that works